ui/lib/core/addon/components/message-error.js

```diff
@@ -11,48 +11,104 @@ import layout from '../templates/components/message-error';
  * <MessageError @model={{model}} />
  * ```
  *
- * @param model=null{DS.Model} - An Ember data model that will be used to bind error statest to the internal
+ * @param {object} [model=null] - An Ember data model that will be used to bind error states to the internal
  * `errors` property.
- * @param errors=null{Array} - An array of error strings to show.
- * @param errorMessage=null{String} - An Error string to display.
+ * @param {array} [errors=null] - An array of error strings to show.
+ * @param {string} [errorMessage=null] - An Error string to display.
  */
-export default Component.extend({
-  layout,
-  model: null,
-  errors: null,
-  errorMessage: null,
-
-  displayErrors: computed(
-    'errorMessage',
-    'model.{isError,adapterError.message,adapterError.errors.@each}',
-    'errors',
-    'errors.[]',
-    function () {
-      const errorMessage = this.errorMessage;
-      const errors = this.errors;
-      const modelIsError = this.model?.isError;
-      if (errorMessage) {
-        return [errorMessage];
-      }
 
-      if (errors && errors.length > 0) {
-        return errors;
-      }
+class MessageError extends Component {
+  get errorMessage() {
```

Comment:
Is this a reasonable application of getters?

Reply:
I would not use a getter unless you're doing something other than just getting the arg (for example, setting a default)